assets/user-profile/components/EditNotificationScheduleModal.tsx

```diff
@@ -49,13 +53,36 @@ class EditNotificationScheduleModalComponent extends React.Component<IProps, ISt
         });
     }
 
+    onSubmitForm(event: React.FormEvent<HTMLFormElement>) {
+        event.stopPropagation();
+        event.preventDefault();
+
+        if (this.formRef.current == null) {
+            return;
```

Comment:
it's better to use `throw new Error('ref missing')` so it's easier to distinguish code that is not supposed to run at all